### PR TITLE
check tooltip if we check popover

### PR DESCRIPTION
**Pull request by Johann-S**
If we check popover we must check tooltip too, because popover inherit from tooltip

### CODE REVIEW

File: src/js/toggler.js [hunk 1]
```diff
@@ -17,6 +23,35 @@ const initToggler = () => {
         }
       })
   })
+
+  checkboxPopper.addEventListener('click', function () {
```
Comment (MartijnCuppens) [inline src/js/toggler.js]: Should't we use change here?
```suggestion
  checkboxPopper.addEventListener('change', function () {
```

**Comment by MartijnCuppens** [comment]
I think we should add more documentation in `toggler.js`, to clarify what's happening

**Comment by Johann-S** [comment]
Hmm 59 lines of code, for me it's not a big file 🤔 
Maybe we can rename with more explicite name our variables or functions, but not a big fan of adding a lot of comments

File: src/js/toggler.js (hunk 1 above)
Comment (Johann-S) [reply src/js/toggler.js]: I'll try and check how it's behave with that change 👍

**Pull request merged**